### ISSUE TITLE
Create rubricator

### ISSUE THREAD

**Issue by noxonsu** [noxonsu/tools.onout.org#50]
Нужно разработать рубрикатор и разместить странички-рыбы по одному в каждой рубрике, что бы собирать трафик и лиды по ключам .

**Comment by VitaliyShulik** [comment]
@noxonsu не совсем понятно, что требуется - давай проясним...
 
Рубрикатор — это маркетинговый документ в виде презентации или текстового файла, который вмещает в себя типы публикаций и их описание.

Нужно создать рубрики для постинга в соц сетях?
Что такое странички-рыбы? Как я понял это сами посты от команды нашей основанные на рубрикаторе? (к примеру: использованию продукта, мнение по поводу рынка и тд?)

**Comment by noxonsu** [comment]
нет, рубрикатор это просто список возможных продуктов разбитых по категориям. пример рубрикатора https://www.appdupe.com/ тут. при клике на любой продукт есть небольшое описание достаточное для того, что бы гугл проиндексировал
